ui/app/models/pki/issuer.js

```diff
@@ -48,6 +48,7 @@ export default class PkiIssuerModel extends Model {
   // parsed from certificate contents in serializer (see parse-pki-cert.js)
   @attr parsedCertificate;
   @attr('string') commonName;
+  @attr('boolean') isRotatable;
```

Comment:
I'd probably rename this and make it readonly, and possible move it up to be near `isDefault`
```suggestion
  @attr isRoot;
```

Reply:
Updated!